src/components/Calculator.js

```diff
@@ -10,42 +10,47 @@ const Calculator = () => {
   };
 
   return (
-    <div className="container">
-      <div className="row-div">
-        <div className="result">
-          { stateObject.total }
-          { stateObject.operation }
-          { stateObject.next }
-        </div>
-      </div>
-      <div className="row-div">
-        <button type="button" className="cell" onClick={onClickHandler}>AC</button>
-        <button type="button" className="cell" onClick={onClickHandler}>+/-</button>
-        <button type="button" className="cell" onClick={onClickHandler}>%</button>
-        <button type="button" className="orange-cell" onClick={onClickHandler}>&#247;</button>
-      </div>
-      <div className="row-div">
-        <button type="button" className="cell" onClick={onClickHandler}>7</button>
-        <button type="button" className="cell" onClick={onClickHandler}>8</button>
-        <button type="button" className="cell" onClick={onClickHandler}>9</button>
-        <button type="button" className="orange-cell" onClick={onClickHandler}>x</button>
+    <div className="main_container">
+      <div className="text">
+        <h2>Let&apos;s do some Math!</h2>
       </div>
-      <div className="row-div">
-        <button type="button" className="cell" onClick={onClickHandler}>4</button>
-        <button type="button" className="cell" onClick={onClickHandler}>5</button>
-        <button type="button" className="cell" onClick={onClickHandler}>6</button>
-        <button type="button" className="orange-cell" onClick={onClickHandler}>-</button>
-      </div>
-      <div className="row-div">
-        <button type="button" className="cell" onClick={onClickHandler}>1</button>
-        <button type="button" className="cell" onClick={onClickHandler}>2</button>
-        <button type="button" className="cell" onClick={onClickHandler}>3</button>
-        <button type="button" className="orange-cell" onClick={onClickHandler}>+</button>
-      </div>
-      <div className="row-div">
-        <button type="button" className="zero-cell" onClick={onClickHandler}>0</button>
-        <button type="button" className="cell" onClick={onClickHandler}>.</button>
-        <button type="button" className="orange-cell" onClick={onClickHandler}>=</button>
+      <div className="container">
+        <div className="row-div">
+          <div className="result">
+            { stateObject.total }
+            { stateObject.operation }
+            { stateObject.next }
+          </div>
+        </div>
+        <div className="row-div">
+          <button type="button" className="cell" onClick={onClickHandler}>AC</button>
+          <button type="button" className="cell" onClick={onClickHandler}>+/-</button>
+          <button type="button" className="cell" onClick={onClickHandler}>%</button>
```

Comment:
- [optional] you can make the size of your calculator buttons and calculator size a little bigger so as to make it more user friendly. great work